src/polyfill.ts

```diff
@@ -215,21 +215,21 @@ export const polyfill = (target: any, options?: PolyfillOptions) => {
 	) as Set<string>
 
 	// expand exclude options using exclusion shorthands
-	for (const excludeOption of excludeOptions) {
+	excludeOptions.forEach(function(excludeOption){
```

Comment:
The current code avoids creating functional scope, which is preferred.

Reply:
I think then it could be just the same but with .values() at the end or explicitly as "something"

Reply:
So, in TS, `excludeOptions` is typed as `Set<string>`. As a kind of [iterable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/@@iterator), `Set` is ideal for this kind of loop.

It’s explicitly demonstrated on [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/@@iterator#using_iterator_with_for..of).

```js
const mySet = new Set();
mySet.add('0');
mySet.add(1);
mySet.add({});

for (const v of mySet) {
  console.log(v);
}
```

Reply:
![image](https://user-images.githubusercontent.com/12800230/147283873-fd3dcf91-fbcd-4a49-a8f6-c5ece4dd93cb.png)
Do you have this flag set to enabled or just perhaps newer typescript version knows that it is good to write that like this?

Reply:
You might have an older TS.